packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/termsOfUse/terms-of-use.html

```diff
@@ -0,0 +1,43 @@
+<p
+    data-ng-bind="::('dedicatedCloud_datacenter_backup_terms_of_use_description1_' + $ctrl.mode) | translate: {
+        datacenter: $ctrl.datacenterId
+    }"
+></p>
+<p
+    data-ng-bind="::('dedicatedCloud_datacenter_backup_terms_of_use_description2_' + $ctrl.mode) | translate"
+></p>
+<strong class="mb-3">
```

Comment:
`strong` is a `display: inline` property so I doubt adding a `.mb-3` will do anything

Reply:
You are right... Changed this tag to a `<p>`